app/styles/_base.scss

```diff
@@ -98,8 +98,28 @@ a {
       top: auto;
     }
   }
+
+  :global &.-underline {
```

Comment:
https://github.com/Vizzuality/GlobalFishingWatch/pull/211/files#diff-da132c0d50c9cd8406b6391530936e2bR84

cant we DRY this?